dist/utils.js

```diff
@@ -213,7 +209,7 @@ angular.module('ts.utils').directive('tsTooltip', function ($templateCache, $tim
             arrowBoxContainer.style.left = leftCommon + element[0].offsetWidth + ARROW_SIZE + parseInt($scope.tsTooltipMargin) + 'px';
             break;
           case 'left':
-            arrowBoxContainer.style.left = leftCommon - $scope.tooltipMain[0].offsetWidth - ARROW_SIZE - parseInt($scope.tsTooltipMargin) + 'px';
+            arrowBoxContainer.style.right = elementOffset.right + ARROW_SIZE + parseInt($scope.tsTooltipMargin) + "px";
```

Comment:
As we discussed at some point, please wrap all the integer stuff to separate from the concatenation operation, e.g. on this line: `arrowBoxContainer.style.right = (elementOffset.right + ARROW_SIZE + parseInt($scope.tsTooltipMargin)) + "px";` Same for other lines here.